test/browser_tests/shared_objects/careersSocialSection.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var _socialSection =
+element( by.css( '[data-qa-hook="social-section"]' ) );
+
+var careersSocialSection = {
+
+  socialSection: _socialSection,
+
+  socialSectionTitles: _socialSection.all( by.css( 'h2' ) ),
+
+  socialSectionDescriptions: _socialSection.all( by.css( '.short-desc' ) ),
+
+  socialSectionLinks: _socialSection.all( by.css( 'a' ) )
+};
+
+module.exports = careersSocialSection;
```

Comment:
I know this is a shared object, but why the change to lowercase in the name compared to the page_objects?